src/dmd/mars.d

```diff
@@ -131,9 +131,9 @@ Where:
   -deps            print module dependencies (imports/file/version/debug/lib)
   -deps=<filename> write module dependencies to filename (only imports)" ~
   "%s" /* placeholder for fpic */ ~ "
-  -dip25           implement http://wiki.dlang.org/DIP25 (experimental)
-  -dip1000         implement http://wiki.dlang.org/DIP1000 (experimental)
-  -dip1008         implement DIP1008 (experimental)
+  -dip25           implement DIP25 http://wiki.dlang.org/DIP25 (experimental)
+  -dip1000         implement DIP1000 https://github.com/dlang/DIPs/blob/master/DIPs/DIP1000.md (experimental)
+  -dip1008         implement DIP1008 https://github.com/dlang/DIPs/blob/master/DIPs/DIP1008.md (experimental)
```

Comment:
The middle `DIP1008` of the three can be removed, because at some point it is pretty obviously `DIP1008`. Same for the other two. The `(experimental)` is also redundant.

Reply:
Ok, Removed redundancies